packages/ingest/package.json

```diff
@@ -6,6 +6,7 @@
     "node": ">=12.18.0"
   },
   "scripts": {
+    "build": "rm -rf dist && mkdir dist && npm run prepare && npm run webpack",
```

Comment:
There is no `npm run webpack` in this package

Reply:
Hm I wonder how this worked when I tested it. Let me fix this, thanks for pointing this out!

Reply:
So I'm not sure how this is working, because there is no `webpack.config.js` in this directory. But you're right that it is working. However, since there is no webpack config for this package, we don't need it. The only compilation needed for this package is the TS compilation. So we should just remove `npm run webpack`

Reply:
That is interesting... Okay, I will remove the `npm run webpack` part of it